ingester/src/server/grpc/rpc_write.rs

```diff
@@ -229,15 +229,23 @@ mod tests {
         column::{SemanticType, Values},
         Column, DatabaseBatch, TableBatch,
     };
+    use lazy_static::lazy_static;
 
     use super::*;
-    use crate::dml_payload::IngestOp;
-    use crate::dml_sink::mock_sink::MockDmlSink;
+    use crate::{
+        dml_payload::IngestOp,
+        test_util::{ARBITRARY_NAMESPACE_ID, ARBITRARY_TABLE_ID},
+    };
+    use crate::{dml_sink::mock_sink::MockDmlSink, test_util::ARBITRARY_PARTITION_KEY};
 
     const NAMESPACE_ID: NamespaceId = NamespaceId::new(42);
     const PARTITION_KEY: &str = "bananas";
     const PERSIST_QUEUE_DEPTH: usize = 42;
 
+    lazy_static! {
+        static ref ALTERNATIVE_TABLE_ID: TableId = TableId::new(76);
+    }
```

Comment:
Same here; `TableId::new` is `const` so this can be:

```suggestion
    const ALTERNATIVE_TABLE_ID: TableId = TableId::new(76);
```